File: IT/1/Strutture dati/Esercizi/Java/Esercizio16.java
/* file Esercizio16.java */

import java.util.Arrays;

public class Esercizio16 {
	public static void main(String [] args) {
		int [] nums = {1, 3, 5, 7, 9};
		int val = 10;

		System.out.println(Arrays.toString(nums));

		for (int i = 0; i < nums.length; i++) {
			for (int j = i+1; j < nums.length; j++) {
				if (nums[i] + nums[j] == val) {
					System.out.println(nums[i] + " + " + nums[j] + " = " + val);
				}
			}
		}
	}
}

File: IT/1/Strutture dati/Esercizi/Python/esercizio16.py
# file esercizio16.py

from __future__ import print_function

nums = [1, 3, 5, 7, 9]
val = 10

print(nums)

for i in range(len(nums)):
	for j in range(i+1, len(nums)):
		if nums[i] + nums[j] == val:
			print("{} + {} = {}".format(nums[i], nums[j], val))

